.github/workflows/backport.yml

```diff
@@ -12,8 +12,17 @@ on:
   # https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
   pull_request_target:
     types: [closed]
-  # See also commands.yml for the /backport triggered variant of this workflow.
+  # See also backport-trigger.yml for the /backport triggered variant of this workflow.
 
 jobs:
   backport:
-    uses: upbound/official-providers-ci/.github/workflows/provider-backport.yml@standard-runners
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: 0
+
+      - name: Open Backport PR
+        uses: zeebe-io/backport-action@be567af183754f6a5d831ae90f648954763f17f5 # v3.1.0
```

Comment:
```suggestion
        uses: zeebe-io/backport-action@be567af183754f6a5d831ae90f648954763f17f5 # v3.1.0
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
          github_workspace: ${{ github.workspace }}
          version: v0.0.8
```

Reply:
These options are no longer needed as the latest version of the action defaults to these values.